theia-full-docker/Dockerfile

```diff
@@ -181,7 +182,8 @@ ARG GITHUB_TOKEN
 RUN yarn --cache-folder ./ycache && rm -rf ./ycache && \
     NODE_OPTIONS="--max_old_space_size=4096" yarn theia build
 EXPOSE 3000
-ENV SHELL /bin/bash
+ENV SHELL /bin/bash\
+    THEIA_DEFAULT_PLUGINS=local-dir:/home/theia/plugins
```

Comment:
This set the `SHELL` var to `/bin/bash THEIA_DEFAULT_PLUGINS=local-dir:/home/theia/plugins`, I think this is a typo and you wanted to define a second env var ?

Reply:
Indeed! 

I think this can be fixed just by replacing the space between `SHELL` and `/bin/bash`  with an `=` , since `ENV` supports defining multiple in one command. 

see doc, towards the middle of the page: https://docs.docker.com/engine/reference/builder/

exerpt:
```
The second form, ENV <key>=<value> ..., allows for multiple variables to be set at one time. Notice that the second form uses the equals sign (=) in the syntax, while the first form does not. Like command line parsing, quotes and backslashes can be used to include spaces within values.

For example:

ENV myName="John Doe" myDog=Rex\ The\ Dog \
    myCat=fluffy
```